docker/addons/mongodb-writer/channels.toml

```diff
@@ -0,0 +1,4 @@
+# If you want to listen on all channels, just pass one element ["*"], otherwise
+# pass the list of channels.
```

Comment:
PLease add to a comment - "list of channels or any other valid NATS subscription regexp"

Reply:
@drasko But this is not true. Writer service always takes everything, and then filters messages using this list and channels field.

Reply:
It's done like this because normalizer publishes messages to `out.senml` topic, so I don't have info about channel as part of NATS subject.

Reply:
Yes, I get it - writers only subscribe ot `out.senml` topic.